source/linear-algebra/source/05-GT/05.ptx

```diff
@@ -0,0 +1,233 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<section xml:id="GT5" xmlns:xi="http://www.w3.org/2001/XInclude">
+    <title>Change of Basis (GT5)</title>
+    <objectives>
+        <ul>
+            <li>
+                <p>
+                    Calculate the change-of-basis matrix between two bases of <m>\IR^n</m>.
+                </p>
+            </li>
+        </ul>
+    </objectives>
+
+<subsection>
+    <title>Warm Up</title>
+    
+</subsection>
+
+<subsection><title>Class Activities</title>
+<remark>
+    <p>
+        So far, when working with the Euclidean vector space <m>\IR^n</m>, we have primarily worked with the standard basis <m>\mathcal{E}=\setList{\vec{e}_1,\dots, \vec{e}_n}</m>.
+        We can explore alternative perspectives more easily if we expand our toolkit to analyze different bases.    
+    </p>
+</remark>
+<activity>
+    <introduction>
+        <p>
+            Let <m>\cal{B}=\setList{\vec{v}_1,\vec{v}_2,\vec{v}_3}=\setList{\begin{bmatrix}1\\0\\1\end{bmatrix},\begin{bmatrix}1\\-1\\1\end{bmatrix},\begin{bmatrix}0\\1\\1\end{bmatrix}}</m>.
```

Comment:
`\setList{\vec{v}_1,\vec{v}_2,\vec{v}_3} ` is producing different subscripts for some reason.

Reply:
thanks (replacing "cal" with "mathcal" fixed it)